internal/transform/config_setting.go

```diff
@@ -0,0 +1,161 @@
+		ContractMaxSizeBytes:            uint32(contractMaxSizeBytes),
+		LedgerMaxInstructions:           int64(ledgerMaxInstructions),
+		TxMaxInstructions:               int64(txMaxInstructions),
+		FeeRatePerInstructionsIncrement: int64(feeRatePerInstructionsIncrement),
+		TxMemoryLimit:                   uint32(txMemoryLimit),
+		LedgerMaxReadLedgerEntries:      uint32(ledgerMaxReadLedgerEntries),
+		LedgerMaxReadBytes:              uint32(ledgerMaxReadBytes),
+		LedgerMaxWriteLedgerEntries:     uint32(ledgerMaxWriteLedgerEntries),
+		LedgerMaxWriteBytes:             uint32(ledgerMaxWriteBytes),
+		TxMaxReadLedgerEntries:          uint32(txMaxReadLedgerEntries),
+		TxMaxReadBytes:                  uint32(txMaxReadBytes),
+		TxMaxWriteLedgerEntries:         uint32(txMaxWriteLedgerEntries),
+		TxMaxWriteBytes:                 uint32(txMaxWriteBytes),
+		FeeReadLedgerEntry:              int64(feeReadLedgerEntry),
+		FeeWriteLedgerEntry:             int64(feeWriteLedgerEntry),
+		FeeRead1Kb:                      int64(feeRead1Kb),
+		FeeWrite1Kb:                     int64(feeWrite1Kb),
+		BucketListSizeBytes:             int64(bucketListSizeBytes),
+		BucketListFeeRateLow:            int64(bucketListFeeRateLow),
+		BucketListFeeRateHigh:           int64(bucketListFeeRateHigh),
+		BucketListGrowthFactor:          uint32(bucketListGrowthFactor),
+		FeeHistorical1Kb:                int64(feeHistorical1Kb),
+		TxMaxExtendedMetaDataSizeBytes:  uint32(txMaxExtendedMetaDataSizeBytes),
+		FeeExtendedMetaData1Kb:          int64(feeExtendedMetaData1Kb),
+		LedgerMaxPropagateSizeBytes:     uint32(ledgerMaxPropagateSizeBytes),
+		TxMaxSizeBytes:                  uint32(txMaxSizeBytes),
+		FeePropagateData1Kb:             int64(feePropagateData1Kb),
+		ContractCostParamsCpuInsns:      contractCostParamsCpuInsns,
+		ContractCostParamsMemBytes:      contractCostParamsMemBytes,
+		ContractDataKeySizeBytes:        uint32(contractDataKeySizeBytes),
+		ContractDataEntrySizeBytes:      uint32(contractDataEntrySizeBytes),
+		MaxEntryExpiration:              uint32(maxEntryExpiration),
+		MinTempEntryExpiration:          uint32(minTempEntryExpiration),
+		MinPersistentEntryExpiration:    uint32(minPersistentEntryExpiration),
+		AutoBumpLedgers:                 uint32(autoBumpLedgers),
+		PersistentRentRateDenominator:   int64(persistentRentRateDenominator),
+		TempRentRateDenominator:         int64(tempRentRateDenominator),
+		MaxEntriesToExpire:              uint32(maxEntriesToExpire),
+		BucketListSizeWindowSampleSize:  uint32(bucketListSizeWindowSampleSize),
+		EvictionScanSize:                uint64(evictionScanSize),
+		LedgerMaxTxCount:                uint32(ledgerMaxTxCount),
+		BucketListSizeWindow:            bucketListSizeWindow,
+		LastModifiedLedger:              uint32(ledgerEntry.LastModifiedLedgerSeq),
+		LedgerEntryChange:               uint32(changeType),
+		Deleted:                         outputDeleted,
```

Comment:
Is it worthwhile to add a boolean for `Expired`? Or do you think the `LedgerEntryChange` is enough?

Reply:
What would the boolean for `Expired` be for? config_settings don't expire afaik